src/components/debounced-input.js

```diff
@@ -29,13 +29,21 @@ function DebouncedInput({ apiRef, timeLimit, elementType: Component, value, onCh
     };
   }
 
+  if (onPressEnter) {
```

Comment:
Ok yeah, of course, this was needed now..

Reply:
Yeah, I followed the `onSearch` example :-)